set.mm

```diff
@@ -661380,13 +661385,85 @@ D Fn ( I ... ( M - 1 ) ) /\
       TVAURIKVCVFVGVDULUNUO $.
   $}
 
+  ${
+    $d R a x $.  $d F a $.  $d M a $.  $d ph a x $.
+    rndrhmcl.r $e |- R = ( N |`s ran F ) $.
+    rndrhmcl.0 $e |- .0. = ( 0g ` N ) $.
+    rndrhmcl.h $e |- ( ph -> F e. ( M RingHom N ) ) $.
+    rndrhmcl.m $e |- ( ph -> M e. DivRing ) $.
+    rndrhmcl.1 $e |- ( ph -> ran F =/= { .0. } ) $.
+    $( The range of a (nontrivial) division ring homomorphism is a division
+       ring.  TODO:  The use of ~ 01eq0ring indicates that ~ df-nzr should come
+       before ~ df-drng .  (Contributed by SN, 17-Feb-2025.) $)
+    rndrhmcl $p |- ( ph -> R e. DivRing ) $=
```

Comment:
Maybe rather the _image_ of a division ring homomorphism? Since we're talking about some linear map here.